app/models/client.rb

```diff
@@ -193,6 +193,12 @@ def clients_with_dupe_contact_info
     Client.after_consent.where(intake: matching_intakes).pluck(:id)
   end
 
+  # We'll consider a client to have completed intake process once any of their
+  # tax returns has passed the intake stage
+  def completed_intake_process?
+    tax_returns.map(&:status_before_type_cast).any? { |status| status >= 200 }
```

Comment:
alternatively 
```ruby
tax_returns.where("status >= ?", 200).exists?
```

Reply:
See comments on portal controller for more concerns about this method of checking doneness